backend/services/performance_service.py

```diff
@@ -0,0 +1,36 @@
+from db.models import Performance, School
+from db import db
+
+def get_performance(id):
+    performance = Performance.query.get(id)
+    return performance and performance.to_dict()
+
+# TODO: debug
+def get_performances(performance_filter=None):
+    """
+    Returns performances with associated event_id. If no event_id is passed in,
+    all performances are returned
+
+    Returns:
+        performances
+    """
+    performances = db.session.query(Performance, School.name) \
+        .outerjoin(School, Performance.school_id == School.id) 
+        
+    if 'event_id' in performance_filter:
```

Comment:
I wasn't sure the best way to use the performance_filter (request parameters) to filter